src/plugins/ui_actions_enhanced/public/drilldowns/url_drilldown/components/url_drilldown_collect_config/i18n.ts

```diff
@@ -57,7 +57,7 @@ export const txtUrlTemplatePreviewLinkText = i18n.translate(
 export const txtUrlTemplateOpenInNewTab = i18n.translate(
   'uiActionsEnhanced.drilldowns.urlDrilldownCollectConfig.openInNewTabLabel',
   {
-    defaultMessage: 'Open in new window',
+    defaultMessage: 'Open URL in new tab',
```

Comment:
I changed the copy here to be more consistent with the dashboard drilldown phrasing; note that this changes the copy for both the navigation embeddable flyout **and** the URL drilldown editor.

| Dashboard drilldowns / links | URL drilldowns / links |
|--------|--------|
| ![image](https://github.com/elastic/kibana/assets/8698078/717208a4-8019-4cc4-8a41-c4b9b2f3eef4) | ![image](https://github.com/elastic/kibana/assets/8698078/841f5c47-a6a8-47d0-9f16-f5f42881e145) |